src/MagentoHackathon/Composer/Magento/DeployManager.php

```diff
@@ -97,7 +100,30 @@ public function doDeploy()
                     $this->io->write($e->getMessage());
                 }
             }
+        }
+    }
+
+    /**
+     * Determine the priority in which the package should be deployed
+     *
+     * @param Entry $package
+     * @return int
+     */
+    private function getPackagePriority(Entry $package)
+    {
+        $result = 100;
+        $maxPriority = max(array_merge($this->sortPriority, [100, 101]));
 
+        if (isset($this->sortPriority[$package->getPackageName()])) {
+            $result = $this->sortPriority[$package->getPackageName()];
+        } elseif (isset($this->highPriority[$package->getPackageName()])) {
+            $packagePriority = $this->highPriority[$package->getPackageName()];
+            $result = intval($maxPriority) + intval($packagePriority);
+        } elseif ($package->getDeployStrategy() instanceof Copy) {
+            $result = 101;
```

Comment:
What is the purpose of this specific value?  Are Copy packages supposed to be before/after packages that have a specific priority?  As written they could be in the middle of packages in the sortPriority array if that array has entries on both sides of 101.  If they're supposed to be last, instead of specifically 101, use min - 1 similar to how you're using max + highPriority.

Reply:
Copy packages should have a higher default priority that the default priority for other packages. So keeping the default priority as 100 and the default for Copy packages as 101 maintains this behavior. A package can define whatever priority they wish so they could have always had it higher priority than Copy packages. So this is different than highPriority and keeping these values maintains the current behavior.